h2o-automl/src/main/java/ai/h2o/automl/AutoMLBuildSpec.java

```diff
@@ -49,6 +50,10 @@ public AutoMLBuildControl() {
     public String project_name = null;
     public String loss = "AUTO";  // TODO: plumb through
     public HyperSpaceSearchCriteria.RandomDiscreteValueSearchCriteria stopping_criteria;
+
+    // Cross-validation fold construction
+    public int nfolds = 5; 
+    //public Model.Parameters.FoldAssignmentScheme fold_assignment;
```

Comment:
Any reason this comment is needed?

Reply:
Yeah I started working on both at the same time, but then decided to comment it out for now.

Reply:
Cool, sounds good.